### PR TITLE
feat: allow to use customized assessment to each criterion in the rubric

### PR DESCRIPTION
**Pull request by BryanttV**
**TL;DR -** [Allow the evaluator to assign a custom range of values]

This PR intends to include a new functionality that allows the evaluator (either student or instructor) to assign each criterion of a rubric a decimal value within a set range.

**What changed?**

In the configuration of each ORA component:
1. Add a check (boolean) that allows to evaluation of each criterion by range. By default, this check will be disabled, and the process will continue to work as it is currently.
2. When the check is active, the button to add options must be disabled, since these will not be needed. Only the button to add criteria will be available.
3. When the check is active, two additional fields (Integer or decimal) should be displayed: minimum value and maximum value, to define the grade range.

In the LMS:
1. All the criteria added to the rubric will be listed, and each of them will contain an input for assigning the personalized assessment.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @BryanttV!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.
